composer.json

```diff
@@ -61,7 +60,6 @@
         "test": [
             "@phpunit",
             "@phpspec"
-        ],
-        "security": "vendor/bin/security-checker security:check"
```

Comment:
Sensiolabs security checker is deprecated.

Reply:
Is there already a recipe (or how is that called in GH) for the new checker? https://github.com/fabpot/local-php-security-checker

Reply:
I am not really into this so you might want to create an issue for this.

Reply:
https://github.com/fabpot/local-php-security-checker/issues/12

Reply:
You could try that out!